lms/djangoapps/branding/admin.py

```diff
@@ -4,6 +4,7 @@
 from django.contrib import admin
 from config_models.admin import ConfigurationModelAdmin
 
-from .models import BrandingInfoConfig
+from .models import BrandingInfoConfig, BrandingApiConfig
 
 admin.site.register(BrandingInfoConfig, ConfigurationModelAdmin)
+admin.site.register(BrandingApiConfig)
```

Comment:
Please use `ConfigurationModelAdmin` as the second argument to this function.  (This prevents deleting configuration in Django admin so we have an audit trail of all changes).